include/openmc/particle.h

```diff
@@ -200,7 +200,17 @@ class Particle {
 
   //! Transport a particle from birth to death
   void transport();
-
+  
+  //! cvmt sampling: continuous varying materials tracking 
+  double sampling_cvmt(Particle *, double);
```

Comment:
We use C++-style declarator layout where `*` or `&` is attached to the type, not the variable name. So this would be `Particle*` instead of `Particle *`. Also, in prototypes, we always put the dummy variable name for clarity.